### PR TITLE
util/mvmap: keep the input order when get multiple values.

### PR DESCRIPTION
**Pull request by coocood**
Some ORM tests assumes the input order when join tables.

### CODE REVIEW

File: util/mvmap/mvmap.go [hunk 1]
```diff
@@ -157,6 +157,11 @@ func (m *MVMap) Get(key []byte) [][]byte {
 		}
 		values = append(values, val)
 	}
+	// Keep the order of input.
```
Comment (shenli) [inline util/mvmap/mvmap.go]: Will this downgrade the performance?

Comment (coocood) [inline util/mvmap/mvmap.go]: This operation is fast, performance change should not be noticeable.

**Comment by hanfei1991** [comment]
I think we don't need to keep the order of join results.

File: util/mvmap/mvmap.go (hunk 1 above)
Comment (shenli) [inline util/mvmap/mvmap.go]: If we have tens of millions rows, it may be a huge cost.

Comment (coocood) [inline util/mvmap/mvmap.go]: Getting one value takes about 400ns, swap a four values array takes about 4ns.
So it adds about 1% operation time for `Get` function.

**Comment by hanfei1991** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**